docs/chapters/baseplatform/building-PELUX-sources.rst

```diff
@@ -41,6 +41,32 @@ Then make repo tool fetch all sources using the command `repo sync`.
     repo init -u https://github.com/Pelagicore/pelux-manifests.git -b <branch>
     repo sync
 
+
+Available images
+^^^^^^^^^^^^^^^^
+
+.. This is to get red and green colours for the symbols below
+.. role:: red
+.. role:: green
+.. raw:: html
+
+    <!-- The roles we defined in rst will translate to style sheet classes -->
+    <style> .red {color:red} .green {color:green} </style>
+
++-------------------------------------+------------+--------------+------------+------------+
+|                                     |      Variant name                                   |
++          Image name                 +------------+--------------+------------+------------+
+|                                     | intel      | intel-qtauto | rpi        | rpi-qtauto |
++=====================================+============+==============+============+============+
+| core-image-pelux-minimal            | :green:`✔` | :green:`✔`   | :green:`✔` | :green:`✔` |
++-------------------------------------+------------+--------------+------------+------------+
+| core-image-pelux-minimal-dev        | :green:`✔` | :green:`✔`   | :green:`✔` | :green:`✔` |
++-------------------------------------+------------+--------------+------------+------------+
+| core-image-pelux-qtauto-neptune     | :red:`✘`   | :green:`✔`   | :red:`✘`   | :green:`✔` |
++-------------------------------------+------------+--------------+------------+------------+
+| core-image-pelux-qtauto-neptune-dev | :red:`✘`   | :green:`✔`   | :red:`✘`   | :green:`✔` |
++-------------------------------------+------------+--------------+------------+------------+
+
```

Comment:
Nice table! I am wondering if we should have the experimental table below or is that something we don't want to point out?

Reply:
I'm not a fan of css-classes which tell how things look like, instead they should be semantically named and say what something is like "available" and "not-available", if we then change the colors some day in the future we only need to change the color, not the class names all over the table.

Reply:
Good point, I will indeed change the names to something more semantic.

Reply:
@rpannek I updated it now with better names!